### PR TITLE
data:antharir.json

### PR DESCRIPTION
**Pull request by antharir**
New entry in data

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3810"></a>

### CODE REVIEW

File: data/RajeshAnthari.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/RajeshAnthari",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/rajesh-anthari-88347480/",
+      "icon": "FaLinkedIn"
```
Comment (Aadarsh805) [inline data/RajeshAnthari.json:16]: Change this icon to `FaLinkedin`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
Change file name to `data/antharir.json`

File: data/RajeshAnthari.json (hunk 1 above)
Comment (antharir) [reply data/RajeshAnthari.json]: updated.
Thanks,

**Review by amandamartin-dev — DISMISSED** [review]
Looks good, just need to make sure your json file name exactly matches your github profile as @Aadarsh805 noted

**Comment by antharir** [comment]
> Looks good, just need to make sure your json file name exactly matches your github profile as @Aadarsh805 noted

Thanks, Amanda. Updated the file name.

**Comment by antharir** [comment]
> Change file name to `data/antharir.json`

Done @Aadarsh805

**Review by antharir — COMMENTED** [review]
Completed change requests.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**